templates/openshift/backup-cronjob-template.yaml

```diff
@@ -20,6 +20,7 @@ objects:
               labels:
                 cronjob-name: ${NAME}
             spec:
+              serviceAccountName: backupjob
```

Comment:
Could this be a param that defaults to "backupjob"?

Reply:
sure, will update